tools/fluent-bit.conf

```diff
@@ -0,0 +1,14 @@
+[INPUT]
+    Name        tail
+    Path        /var/log/*.log
+[Output]
+    Name loki
+    Match *
+    Url http://host.docker.internal:3100/api/prom/push
```

Comment:
remove in favor of new path?

Reply:
yes I rebase to master also to get the latest endpoints.